tests/kafkatest/services/security/minikdc.py

```diff
@@ -38,9 +38,10 @@ class MiniKdc(Service):
     LOCAL_KEYTAB_FILE = "/tmp/keytab"
     LOCAL_KRB5CONF_FILE = "/tmp/krb5.conf"
 
-    def __init__(self, context, kafka_nodes):
+    def __init__(self, context, kafka_nodes, extra_principals = ""):
```

Comment:
should `extra_principals` be a string or a list of individual principals (i.e. a list of strings)?